code/datums/brain_damage/severe.dm

```diff
@@ -117,6 +118,11 @@
 		owner.remove_trait(X, "trauma_paralysis")
 	owner.update_disabled_bodyparts()
 
+/datum/brain_trauma/severe/paralysis/paraplegic
+	random_gain = FALSE
+	paralysis_type = "legs"
+	resilience = TRAUMA_RESILIENCE_ABSOLUTE
```

Comment:
i think you have to blacklist this to avoid it poping up in the random trauma pool

Reply:
That's what random_gain does.

Reply:
oh i forgot, my bad